modules/configuring-access-control-and-sending-invitations.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * configuration/inviting-users-to-rhacs.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-access-control-and-sending-invitations_{context}"]
+= Configuring access control and sending invitations
+
+.Procedure
+
+. To invite a user, use one of the following methods:
+* In the {product-title-short} portal, navigate to *Platform Configuration -> Access Control -> Auth providers* tab, and then click *Invite users*.
+* Navigate to your profile icon, and then click *Invite users*.
+
+. In the *Invite users* dialog box, provide the following information:
+* *Emails to invite*: Enter one or more email addresses of the users you want to invite. Ensure that they are valid email addresses associated with the intended recipients.
+* *Provider*: From the drop-down list, select a provider you want to use for each invited user.
++
+[IMPORTANT]
+====
+* If you have only one authentication provider available, it is selected by default.
+* If multiple authentication providers are available and at least one of them is `Red Hat SSO` or `Default Internal SSO`, that provider is selected by default.
+* If multiple authentication providers are available, but none of them is `Red Hat SSO` or `Default Internal SSO`, you are prompted to select one manually.
+* If you have not yet set up authentication providers, you will receive a warning message; the form will be disabled and you will receive a link that takes you to the *Access Control* section where you can configure an authentication provider.
```

Comment:
I'd remove the semi-colon. I think you could keep these as separate sentences.
Try to remove future tense
Some other suggestions to consider

```suggestion
* If you have not yet set up an authentication provider, a warning message appears and the form is disabled. Click the link, which takes you to the *Access Control* section to configure an authentication provider.
```